packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -334,33 +336,40 @@ class EnrollVerbHandler extends AbstractVerbHandler {
         await getEnrollDataStoreValue(enrollmentKey);
 
     if (_doesEnrollmentHaveManageNamespace(enrollDataStoreValue)) {
-      await _fetchAllEnrollments(enrollmentKeysList, enrollmentRequestsMap);
+      await _fetchAllEnrollments(enrollmentKeysList, enrollmentRequestsMap,
+          enrollVerbParams?.enrollmentStatusFilter);
     } else {
       if (enrollDataStoreValue.approval!.state !=
           EnrollmentStatus.expired.name) {
         enrollmentRequestsMap[enrollmentKey] = {
           'appName': enrollDataStoreValue.appName,
           'deviceName': enrollDataStoreValue.deviceName,
           'namespace': enrollDataStoreValue.namespaces,
-          'encryptedAPKAMSymmetricKey' : enrollDataStoreValue.encryptedAPKAMSymmetricKey
+          'encryptedAPKAMSymmetricKey':
+              enrollDataStoreValue.encryptedAPKAMSymmetricKey
         };
       }
     }
     return jsonEncode(enrollmentRequestsMap);
   }
 
-  Future<void> _fetchAllEnrollments(List<String> enrollmentKeysList,
-      Map<String, Map<String, dynamic>> enrollmentRequestsMap) async {
+  Future<void> _fetchAllEnrollments(
+      List<String> enrollmentKeysList,
+      Map<String, Map<String, dynamic>> enrollmentRequestsMap,
+      List<EnrollmentStatus>? enrollmentStatusFilter) async {
```

Comment:
add enrollmentStatusFilter as optional param

Reply:
change has been made as part of [fix: make verbParams optional param from fetchEnrollments()](https://github.com/atsign-foundation/at_server/pull/1846/commits/ffebc36ffdb07cc4133207cbdb68c9063c6aa7e0)